src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,125 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px;
+
+  &__row {
+    display: flex;
+    margin-bottom: 46px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:not(:last-child) {
+    margin-right: 48px;
+  }
+
+  &__image {
+    padding: 16px 3px 0;
```

Comment:
please, move this padding-top to card's padding

Reply:
done

Reply:
but:
![image](https://user-images.githubusercontent.com/126285904/230596437-c5d6bf6f-0bcd-424a-93ae-de17021fd227.png)